src/test/tests.ts

```diff
@@ -606,12 +597,16 @@ describe('Client', function () {
   it('should handle errors that are not an array', function (done) {
     const errorMessage = 'Just an error';
     const payload = {
-      errors: {message: errorMessage},
+      message: errorMessage,
     };
     testBadServer(payload, errorMessage, done);
   });
 
   it('should throw an error when the susbcription times out', function (done) {
+    // TODO
+    // This test is no longer needed. Do u agree Hagaico?
```

Comment:
Because the code we have in the old client to manage the waiting subscriptions was removed, do u remember @DxCx ?

That was the old code:

```javascript
setTimeout( () => {
      if (this.waitingSubscriptions[subId]) {
        handler([new Error('Subscription timed out - no response from server')]);
        this.unsubscribe(subId);
      }
    }, this.subscriptionTimeout);
```

Now we control the timeout in a connection wise way, so this test is no longer needed right?